### PR TITLE
Bump revision to 0.12.0

### PR DESCRIPTION
**Pull request by jkeiser**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,2 +1,3 @@
 source "https://rubygems.org"
+gemfile
```
Comment (tyler-ball) [inline Gemfile]: Why is this needed _inside_ the Gemfile itself?

Comment (jkeiser) [inline Gemfile]: Copied it from elsewhere, probably not needed?
